pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -81,6 +83,25 @@ let
             ++ lib.optional (config.pulseaudio or true) libpulseaudio;
       gtk_modules = [ libcanberra-gtk2 ];
 
+      enterprisePolicies = {
+        policies = extraPolicies;
+      };
+
+      policiesJson = builtins.toFile "policies.json"
+        (builtins.toJSON enterprisePolicies);
+
+      mozillaCfg = builtins.toFile "mozilla.cfg" ''
+        // First line must be a comment
+        // User customization
+        ${extraPrefs}
+      '';
+
+      firefox-user-config = runCommand "firefox-user-config" {} ''
+        mkdir $out
+        cat > "$out/policies.json" < ${policiesJson}
+        cat > "$out/mozilla.cfg" < ${mozillaCfg}
```

Comment:
Why not just `cp ${policiesJson} $out/policies.json`?

Reply:
For no reason in particular. The two are mostly interchangeable.
If `cp` is preferred over `cat`, I can change it.
Depending on your viewpoint `cp` is the more complex tool I guess.

Reply:
I think we should use `cp` to copy files :)

Reply:
Ok :-)